frontend/src/lib/geo.js

```diff
@@ -10,7 +10,7 @@ const tile = {
   attribution: "Tiles &copy; Esri &mdash; DeLorme, NAVTEQ, Esri",
 };
 
-const getBounds = (administration) => {
+export const getBounds = (administration) => {
```

Comment:
Why do you need to export this? getBounds is already part of geo.

geo.getBounds()